src/app/scripts/cac/urlrouting/cac-urlrouting.js

```diff
@@ -14,8 +14,8 @@ CAC.UrlRouting.UrlRouter = (function (_, $, UserPreferences, Utils, Navigo) {
     var SHARED_READ = ['maxWalk', 'wheelchair', 'bikeTriangle'];
     var EXPLORE_ENCODE = SHARED_ENCODE.concat(['placeId', 'exploreTime']);
     var EXPLORE_READ = EXPLORE_ENCODE.concat(SHARED_READ);
-    var DIRECTIONS_ENCODE = SHARED_ENCODE.concat(['destination', 'destinationText', 'waypoints']);
-    var DIRECTIONS_READ = DIRECTIONS_ENCODE.concat(SHARED_READ).concat(['arriveBy']);
+    var DIRECTIONS_ENCODE = SHARED_ENCODE.concat(['destination', 'destinationText']);
+    var DIRECTIONS_READ = DIRECTIONS_ENCODE.concat(SHARED_READ).concat(['arriveBy', 'waypoints']);
```

Comment:
I agree that that sounds like a sensible and desirable thing to do, but I don't think we're set up for it.  I would have to dig in a little more to be more specific, but my memory of how this is all set up is that local storage is authoritative because there are lots of components that interact with it and aren't set up to take their parameters from arguments, so the way the app loads directions from a URL is to save all the URL params to local storage then kick off the directions generation function which reads from local storage to do its thing.

So I'm for the existence of a category of operational parameters distinct from the parameters in storage, but my understanding is that we're a non-trivial refactor away from having that.